test/integration/build_admission_test.go

```diff
@@ -53,6 +54,9 @@ func TestPolicyBasedRestrictionOfBuildStrategies(t *testing.T) {
 		t.Errorf("unexpected error: %v", err)
 	}
 
+	// Give the policy cache a chance to catch up
+	time.Sleep(time.Second)
```

Comment:
if we use the same wait time elsewhere for the policy cache, it may be useful to make it a constant in test.util in case we ever need to bump it up.